FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Common/Configurations/FilterLocation.cs

```diff
@@ -0,0 +1,30 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using Newtonsoft.Json;
+
+namespace Microsoft.Health.Fhir.Synapse.Common.Configurations
+{
+    public class FilterLocation
```

Comment:
This options is meant to share the same configuration section `filter` with `FilterConfiguration` options, right?

Reply:
yes, parsed from the same "filter" field.